icepyx/tests/test_spatial.py

```diff
@@ -0,0 +1,262 @@
+import pytest
+import warnings
+import datetime as dt
+import numpy as np
+from shapely.geometry import Polygon
```

Comment:
We try to follow the abc order convention for imports of common packages - could you please update this?

Reply:
updated with most recent push